### ISSUE TITLE
Parallel Processing with Nightwatch without Selenium

### ISSUE THREAD

**Issue by cortezcristian** [TestArmada/magellan#239]
Is it possible to use this framework without java and selenium?
Using nightwatch parallel auto feature directly avoiding selenium? Or is selenium a core component of Magellan?

**Comment by archlichking** [comment]
Nope, selenium isn't a must to the magellan eco-system. There is an option to disable selenium auto launch in nightwatch.json. Similarly like http://testarmada.io/#manageAppiumLifecycle, `selenium.start_process` can be false so that nightwatch won't automatically launch selenium server for you.

**Comment by cortezcristian** [comment]
I'm running a local executor, if i set `start_process: false,` obviously it start failing.

<img src="https://user-images.githubusercontent.com/1882205/29567257-fd88e86c-8722-11e7-994f-42daaa6248dd.png" width="350" />

Where should be the place to start the chromedriver server? I mean calling chromedriver.start/stop in magellan? This is from the official guide: http://nightwatchjs.org/gettingstarted under ChromeDriver section 3.

![image](https://user-images.githubusercontent.com/1882205/29567604-30ebd4a2-8724-11e7-9a33-afb31f0289bd.png)

**Comment by archlichking** [comment]
i'm afraid you need to follow all the instructions especially this one
```
We also need to clear the default_path_prefix, as it is set by default to /wd/hub, which is what selenium is using.
```

**Comment by cortezcristian** [comment]
I did set `"default_path_prefix": "",` to be empty under `test_settings/default`. Still failing

![image](https://user-images.githubusercontent.com/1882205/29579383-189f269a-8749-11e7-974f-6eadb37ea3ba.png)

**Comment by archlichking** [comment]
when you see `error retrieving a new session ...`, it means nightwatch is still looking for selenium server. you may still have selenium.start_process enabled somewhere. use `--debug` to have local executor to output the actual desiredCapabilities.

**Comment by cortezcristian** [comment]
I did that:
```
[DEBUG][Local Executor] executor config: {
	"desiredCapabilities": {
		"browserName": "chrome",
		"chromeOptions": {
			"args": ["--no-sandbox", "start-maximized", "disable-web-security"],
			"prefs": {
				"credentials_enable_service": false,
				"profile.password_manager_enabled": false,
				"profile.managed_default_content_settings.images": 2
			}
		},
		"acceptSslCerts": true
	}
}
```
